risc0/zkvm/Cargo.toml

```diff
@@ -128,6 +128,7 @@ prove = [
 std = [
   "anyhow/std",
   "num-traits?/std",
+  "dep:risc0-binfmt",
```

Comment:
I think we should avoid using `std` as a way to pull in deps. It should only be used for forwarding the `std` feature flag to other deps.

Reply:
removed